module-04/02-Exporting-Custom-Templates-Template-Parts.md

```diff
@@ -83,38 +83,11 @@ Then you can register your custom template, giving it a name, a list of postType
 
 ## A little housekeeping
 
-If you switch back to the Site Editor, and refresh the Templates list, you'll see there are now two Page Alternative templates listed. 
+If you switch back to the Site Editor, and refresh the Templates list, you'll see there is now two Page Alternative template listed. 
 
 ![Duplicate Templates](https://learn.wordpress.org/files/2022/10/updated-custom-templates-list.png)
 
-One is marked as added by your user, and the other as added by the theme. The one added by the user was created soley in the editor, and is therefore only stored in the database. The other is the file you just created and registered. You can therefore delete the one added by the user.
-
-To practice this process, export the post alternative template you created in the previous lesson. Follow the same steps as you did for the alternative page
-
-1. Export the template to the relevant template file
-2. Register the template in theme.json
-4. Delete the template added by the user
-
-Remember to add the new custom template to the `customTemplates` array in the theme.json file, using a comma between each custom template object. Remember to also set the correct post type for the alternative post template
-
-```json 
-"customTemplates": [
-		{
-			"name": "page-alternative",
-			"postTypes": [
-				"page"
-			],
-			"title": "Page Alternative"
-		}, 
-		{
-			"name": "post-alternative",
-			"postTypes": [
-				"post"
-			],
-			"title": "Post Alternative"
-		},
-	],
-```
+When adding new custom templates, remember to add it to the `customTemplates` array in the `theme.json` file using a comma between each custom template object. Remember to also set the correct post type for each custom template.
```

Comment:
Hi @cynthianorman I noticed that you've removed quite a chunk of instructions here as it relates to custom templates. I'm curious if these instructions are no longer relevant, or if this was a mistake?

Reply:
@cynthianorman my apologies, I just saw your comment on our [main tracking issue](https://github.com/WordPress/Learn/issues/1794).